data/json/items/classes/book.json

```diff
@@ -25,7 +25,10 @@
     "symbol": "?",
     "color": "green",
     "use_action": "MA_MANUAL",
-    "intelligence" : 9,	
-    "book_data": { "time": 10 }
+    "book_data":
```

Comment:
The jobs raising errors is due to something in this file according to the job log (`make: *** [data/json/items/classes/book.json] Error 1`), and I think it's because the formatting is off. I think what might work is:

```
"book_data": {
    "time": 10,
    "intelligence: 9
}
```

Assuming of course that the problem really is just the formatting.